src/Umbraco.Web.UI.Client/src/less/canvas-designer.less

```diff
@@ -270,10 +270,10 @@ a, a:hover{
     height: 100%;
     padding: 0;
     margin-left: -80px;
-    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-family: "Lato", Helvetica, Arial, sans-serif;
```

Comment:
In theory could this entire chain of font's be moved to a less variable (in an ideal world)
If this is too big of task then, we can do it another time & for another task